Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM debian:bullseye-slim AS base
 
+RUN update-ca-certificates
+
```

Comment:
Will this work if ca-certificates aren't installed? And don't we have to do this on the final container (e.g. after line 13)?

Reply:
Sadly, no Docker in CI I don't think 😬

Reply:
The first container is the base container, so running it in there should trickle into the others.....I need to go find another laptop to check it...ill report back